lib/mimdb_web/router.ex

```diff
@@ -17,6 +17,7 @@ defmodule MimdbWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+    resources "/genres", GenreController
```

Comment:
If you decide to remove show, you can alter this line to _not_ generate the show route

Reply:
added ", except [:show]" to end of line 20